packages/app/src/views/Home.vue

```diff
@@ -70,10 +102,34 @@ export default defineComponent({
 
   setup() {
     const groups = ref<NFCTagGroup[]>([]);
+    const selectedTag = ref<NFCTag | null>(null);
 
-    onMounted(async () => {
-      const res = (await feathers.service('nfc-tags').find()) as Paginated<NFCTag>;
+    provide('selectedTag', selectedTag);
+
+    const toggleTag = (tag: NFCTag): void => {
+      selectedTag.value = tag === selectedTag.value ? null : tag;
+      buttonTransitionActive.value = true;
```

Comment:
Could you put that into the template as you did it with `@leave` for `infoTransitionActive`.